projects/e2e/workbench/src/page-object/testcase-bb9700a6-view.po.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2018-2019 Swiss Federal Railways
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ *  SPDX-License-Identifier: EPL-2.0
+ */
+
+import { $ } from 'protractor';
+import { clickElement } from '../util/testing.util';
+
+export class TestcaseBb9700a6ViewPO {
+
+  private _linkFinder = $('a.e2e-view-navigation-link');
```

Comment:
In workbench e2e-tests it is required to use a component finder upfront. This is different to tests written for the platform because not in the scope of a separate browsing context given by the iframe. Please see the PO for the welcome page.

Reply:
done